pkg/kubecost/coverage.go

```diff
@@ -0,0 +1,108 @@
+package kubecost
+
+import (
+	"time"
+
+	"github.com/opencost/opencost/pkg/filter"
+)
+
+// Coverage This is a placeholder struct which can be replaced by a more specific implementation later
+type Coverage struct {
```

Comment:
A good docstring here would be great, but maybe you're waiting to discuss more in depth?